### PR TITLE
Added bmc creds upgrade note

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

**Comment by ekohl** [comment]
There is a merge conflict, but 3.0 is about to go GA. I think this needs to be merged before that. Would you mind rebasing?

**Comment by lzap** [comment]
I will say it again, the way we handle release and ugprade notes is abysmal and I am that close to ignoring this process completely. I have been fighting with rebasing, branches and conflicts for years, this must stop.

@upadhyeammit I suggest to keep these notes in a Redmine or Discourse (wiki) until a release is ready, that is a one-click edit instead of this.

Rebased.

File: _includes/manuals/3.0/1.2_release_notes.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Fact parsers for Ansible, Chef, Salt and Subscription Manager were moved to the
 * The `host_param_true?` and `host_param_false?` macros changed the meaning of the `default_value` parameter. The value is now used for the comparison if the parameter is not set on the host, instead of returing the value directly. Please update your templates accordingly.
 * The Debian packages are signed with a new key ([5B7C3E5A735BCB4D615829DC0BDDA991FD7AAC8A](/static/keys/5B7C3E5A735BCB4D615829DC0BDDA991FD7AAC8A.pub)). Please import it to verify the packages.
 * Foreman 2.5 for Debian/Ubuntu had briefly wrong versioned Ansible and Remote Execution plugins published, please read our [blog post how to properly upgrade these](/2021/08/please-manually-upgrade-ansible-and-remote-execution-plugins-on-25.html).
+* Setting named "BMC credentials access", which controls BMC credentails access through ENC YAML output and templates, is turned off by default for increased security. To turn it on, go to Administer - Settings.
```
Comment (ekohl) [inline _includes/manuals/3.0/1.2_release_notes.md]: Perhaps we can also mention the exact setting name which must be changed.

Comment (lzap) [reply _includes/manuals/3.0/1.2_release_notes.md]: What do you mean, I did mention exactly that:

```
$ ag "BMC credentials access"
app/models/setting/auth.rb
29:      set('bmc_credentials_accessible', N_("Permits access to BMC interface passwords through ENC YAML output and in templates"), true, N_('BMC credentials access')),
```

Do you mean to mention Authentication tab?

Comment (ekohl) [reply _includes/manuals/3.0/1.2_release_notes.md]: Ah right, you did. I didn't think about about the first sentence so I found it hard to connect. How about:
```suggestion
* BMC credentials access through ENC YAML output and templates is turned off by default for increased security. To turn it on, go to Administer - Settings and turn the "BMC credentials access" setting on.
```

Comment (lzap) [reply _includes/manuals/3.0/1.2_release_notes.md]: Sure, amended.

**Review by ekohl — APPROVED** [review]
Thanks!

**Comment by ekohl** [comment]
So now I remember why this wasn't merged: the core PR hasn't been merged. https://github.com/theforeman/foreman/pull/8359 was even closed for inactivity.

**Comment by lzap** [comment]
For the record this was yanked before the final release. I have created a new PR.

**Pull request merged**